src/main/java/core/basesyntax/suppliers/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.suppliers;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        return String.valueOf(Color.values()[index]);
```

Comment:
[Common mistackes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#use-name-for-getting-string-representation-of-enum-constants)
**Use name() for getting String representation of enum constants**
Don’t use toString() or String.valueOf()(it will call toString() under the hood) for getting String representation of enum constants.